cli/src/cli_util.rs

```diff
@@ -2135,6 +2137,129 @@ fn load_template_aliases(
     Ok(aliases_map)
 }
 
+pub fn get_git_repo(store: &Store) -> Result<git2::Repository, CommandError> {
```

Comment:
I wonder if some of these should instead go into a new `git_util.rs`, but we can leave that for later.

Reply:
I thought about this, since we're accumulating `git` functions in `cli_utils`. I can do that refactoring afterwords or as this PR is slimmed down. (I might split out the first two commits.)